SPECS/nodejs/nodejs.spec

```diff
@@ -17,6 +17,8 @@ Requires:       coreutils >= 8.22
 Requires:       openssl >= 1.0.1
 Requires:       python3
 
+Provides: npm
```

Comment:
Please also set the version and release number for `npm` similarly to what Fedora does for instance. From Fedora's `nodejs.spec`:
```
# npm - from deps/npm/package.json
%global npm_epoch 1
%global npm_major 6
%global npm_minor 14
%global npm_patch 12
%global npm_version %{npm_major}.%{npm_minor}.%{npm_patch}
(...)
Provides: npm = %{npm_epoch}:%{npm_version}
```

We can skip the epoch in our case, though.